src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- Перша навігаційна панель -->
+    <header>
+      <div class="logo-container">
+        <a
+          class="imgLogo logo-1200"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
```

Comment:
The `alt` attribute should describe the image content. Instead of 'Logo', describe what the logo is about, for example 'Company X Logo'.